src/scripts/crowdin-import.js

```diff
@@ -0,0 +1,369 @@
+const { resolve, join } = require("path")
+
+/**
+ * 0. Console summary flags
+ */
+const VERBOSE = false
+const FULL_BUCKET_NAME_SUMMARY = false
+
+/**
+ * 1. Copy languages folder from Crowdin export to ./.crowdin, or enter override
+ * to point to local unzipped download folder
+ * ie: /Home/alice/Downloads/Ethereum.org Translations (translations)/
+ */
+const CROWDIN_ROOT_OVERRIDE = ""
+
+/**
+ * 2. Select languages and buckets to import by adding the number of the
+ * corresponding content bucket to the `buckets` array
+ * ie. `buckets: [1, 10],` would import the "Homepage" and "Learn" buckets
+ */
+const SELECTION = [
+  {
+    lang: "ar",
+    buckets: [],
+  },
+  {
+    lang: "az",
+    buckets: [],
+  },
+  {
+    lang: "bg",
+    buckets: [],
+  },
+  {
+    lang: "bn",
+    buckets: [],
+  },
+  {
+    lang: "ca",
+    buckets: [],
+  },
+  {
+    lang: "cs",
+    buckets: [],
+  },
+  {
+    lang: "da",
+    buckets: [],
+  },
+  {
+    lang: "de",
+    buckets: [],
+  },
+  {
+    lang: "el",
+    buckets: [],
+  },
+  {
+    lang: "es",
+    buckets: [],
+  },
+  {
+    lang: "fa",
+    buckets: [],
+  },
+  {
+    lang: "fi",
+    buckets: [],
+  },
+  {
+    lang: "fr",
+    buckets: [],
+  },
+  {
+    lang: "gl",
+    buckets: [],
+  },
+  {
+    lang: "hi",
+    buckets: [],
+  },
+  {
+    lang: "hr",
+    buckets: [],
+  },
+  {
+    lang: "hu",
+    buckets: [],
+  },
+  {
+    lang: "id",
+    buckets: [],
+  },
+  {
+    lang: "ig",
+    buckets: [],
+  },
+  {
+    lang: "it",
+    buckets: [],
+  },
+  {
+    lang: "ja",
+    buckets: [],
+  },
+  {
+    lang: "ka",
+    buckets: [],
+  },
+  {
+    lang: "kk",
+    buckets: [],
+  },
+  {
+    lang: "ko",
+    buckets: [],
+  },
+  {
+    lang: "lt",
+    buckets: [],
+  },
+  {
+    lang: "ml",
+    buckets: [],
+  },
+  {
+    lang: "ms",
+    buckets: [],
+  },
+  {
+    lang: "mr",
+    buckets: [],
+  },
+  {
+    lang: "nb",
+    buckets: [],
+  },
+  {
+    lang: "nl",
+    buckets: [],
+  },
+  {
+    lang: "ph",
+    buckets: [],
+  },
+  {
+    lang: "pl",
+    buckets: [],
+  },
+  {
+    lang: "pt",
+    buckets: [],
+  },
+  {
+    lang: "pt-br",
+    buckets: [],
+  },
+  {
+    lang: "ro",
+    buckets: [],
+  },
+  {
+    lang: "ru",
+    buckets: [],
+  },
+  {
+    lang: "se",
+    buckets: [],
+  },
+  {
+    lang: "sk",
+    buckets: [],
+  },
+  {
+    lang: "sl",
+    buckets: [],
+  },
+  {
+    lang: "sr",
+    buckets: [],
+  },
+  {
+    lang: "sw",
+    buckets: [],
+  },
+  {
+    lang: "th",
+    buckets: [],
+  },
+  {
+    lang: "tr",
+    buckets: [],
+  },
+  {
+    lang: "uk",
+    buckets: [],
+  },
+  {
+    lang: "vi",
+    buckets: [],
+  },
+  {
+    lang: "zh",
+    buckets: [],
+  },
+  {
+    lang: "zh-tw",
+    buckets: [],
+  },
+]
+/**
+ * 3. Execute script by running `yarn crowdin-import`
+ */
+
+/******************************
+ * Primary script logic below *
```

Comment:
Would be nice to use this block to explain how to use the script (in case we add arguments) and what the script does from a black box perspective (inputs & outputs).